source/documentation/contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,100 @@
+[[contributing-user-stories]]
+= Contribute user stories to oVirt documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about how to create user stories for oVirt GitHub repository
+
+toc::[]
+
+== Modularization backstory
+oVirt docs are modularized.
+All existing content has been replaced with content that is based on user stories and
+complies with the modularization guidelines. All future content must both
+support a user story and be modular.
+
+== How do I contribute modularized content?
+To contribute modularized content, you need to write a user story, create
+documentation modules to support the user story, and create an assembly for the
+story.
+
+== What if I don't want to write in modules?
+If you don't want to write the modules yourself but have a content change,
+write a user story, provide details to support the story, and reach out to the
+oVirt docs team.
+
+== How do I write a user story? Is there a template?
+Instead of a template, we have a series of questions for you to answer to
+create the user story. Follow the same steps if you are writing the modules
+yourself or if you plan to work with the docs team.
+
+The basic format of a user story is:
+
+----
+As a <type of user>, I want to <goal state> because <reason behind the goal>.
+----
+
+For example, "As an oVirt administrator, I want to migrate the Data Warehouse to a remote machine because I want to achieve better performance for the Engine machine."
+
+Use the following questions to guide you in providing the context for your user story and the necessary technical details to start a draft.
+You don't have to answer all of these questions, only the ones that make sense to your particular user story.
+
+=== Feature info
+* What is the feature being developed? What does it do?
+* How does it work?
+* Are there any configuration files/settings/parameters being added or modified? Are any new commands being added or modified?
+* What tools or software does the docs team need to test how this feature works? Does the docs team need to update any installed software?
```

Comment:
Recommendation: "documentation team" instead of "docs team"

Reply:
Done.